config/locales/en.yml

```diff
@@ -4,7 +4,8 @@ en:
   label_full_text_search_result_order_target: Sort by
   label_full_text_search_result_order_type: Sort order
   label_full_text_search_order_target_score: score
-  label_full_text_search_order_target_date: updated at
+  label_full_text_search_order_target_last_modified_at: updated at
+  label_full_text_search_order_target_registered_at: created at
```

Comment:
How about using `last modified time` and `registered time` (or something)?
I'm not sure "XXX at" is a natural label for English speaker.

Reply:
fix: [21c083d](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/21c083da9ecc4c7d75ba8c8a35c4a0b33f7452d0)

I have aligned the verbs in the past tense, referencing the services mentioned below.
| Example 1 (GitHub) | Example 2(Google Drive) |
| --- | --- |
| <img src=https://github.com/clear-code/redmine_full_text_search/assets/45173523/e66b87d9-3f5a-4c34-aec1-3568d119c26c  width=100%> | <img src=https://github.com/clear-code/redmine_full_text_search/assets/45173523/20bedee8-05e2-4ed0-8e46-d49d1c726337  width=100%> |

Reply:
Or using `updated` or `created` simply might also be a good idea because some pages use the `Created` or `Updated`.